app/styles/app/layouts/insights.scss

```diff
@@ -1,339 +1,341 @@
-$axis-line-color: $dry-cement;

```

Comment:
Ugh, so I guess some of these files had CRLF line endings. I did some dev through Windows Subsystem for Linux, and I thought I had my editor set to always do LF line endings, but I guess not. Anyway, I've recently altered my setup such that I'm going to always be using my macbook. And I'm thinking I'll open up a PR to convert any other files that have CRLF line endings.

Reply:
How did it go with WSL? The last attempt I found source maps to not being working properly because of different filesystem root (I guess it was something like `/c/something` inside VM vs `C:\something` outside)

Reply:
It went fine, a lot better than previous experiences I've had developing non-windows stuff on windows hah. I didn't notice a source maps issue, but I wasn't paying close attention to it and that makes sense, I know it does some weird stuff with filesystem paths.

The problem that I ran into is that it was slow, at least for what we do.  Ember CLI build/serve took a much longer time to finish than on OSX/linux. I've heard that filesystem operations can be slow on WSL, especially if there are a lot of them at once.

Reply:
Did you try this approach for Windows builds performance improvement https://cli.emberjs.com/release/appendix/windows/ ? Just enabling symlinks makes it much better, even close to Mac/Linux performance. The reason for that is that Broccoli (Ember CLI build pipeline tool) heavily uses filesystem. Basically, every single build step is a copy of the app and dependencies. On POSIX-compatible filesystems it uses symlinks, which are fast and transparent. But in Windows symlinks are not available without some magic, thus Broccoli has to COPY the entire app on each step. Also in this case `tmp` folder may take several Gb of space.

Reply:
Oooh, neat! I tried a few different things, but I don't remember doing anything with symlinks. Maybe I'll give those suggestions a try tomorrow 😄